### DISCUSSION TITLE
How to create users trough SQL script?

### DISCUSSION THREAD

**Discussion by afonsolopez** [uxbox/uxbox#2432] [Q&A] [answered]
Hello guys, can you help me with a SQL script to register a new user directly into Postgress?

I've tried a couple of times but I'm stuck at generating the hash password for my users on the SQL script.

**Comment by niwinz** [comment, marked as answer]
Unfortunatelly, right now you don't have an option for generate the hash outside the penpot backend code. A good workaround is probably, is copy an already known hash from existing user, and reuse it. Then, users can change the password on the profile.

**Reply by afonsolopez** [reply]
Got it! Thank you!
Btw there's no way to generate a  argo2id like hash for this temporary password?

**Reply by niwinz** [reply]
you can do it, but you need to reimplement the same logic as it here: https://github.com/funcool/buddy-hashers/blob/master/src/clj/buddy/hashers.clj#L174 in the language you are using
